CMSgov/mdct-mcr
# collapse page templates into PageTemplate; rename McparReportPage to ReportPage

## Description

Because the component formerly known as `McparReportPage` is now 100% report agnostic (ready for MLR and NAAAR woooo), it doesn't make much sense to keep that name. But we already had a page template named `ReportPage`, so instead of getting fancy with names I collapsed `ReportPage` into `BasicPage` and renamed them `PageTemplate`. Then renamed `McparReportPage` → `ReportPage`. Makes sense, right?

### How to test

All the stuff should still be working and looking fine. Really just visit the routes and see that it's 👌 .

### Changed Dependencies

n/a

## Code author checklist

## Reviewer checklist (two different people)
